jupyterhub/templates/hub/configmap.yaml

```diff
@@ -19,8 +19,7 @@ data:
 {{- /* copy values.singleuser */ -}}
 {{- $_ := set $values "singleuser" (omit .Values.singleuser "imagePullSecret") }}
 {{- $_ := set $values.singleuser "imagePullSecret" (omit .Values.singleuser.imagePullSecret "password") }}
+{{- /* preserve behavior of deprecated hub.extraConfigMap */ -}}
+{{- $_ := set $values "custom" (merge .Values.hub.extraConfigMap $values.custom) }}
```

Comment:
Hmmm, this is so complicated but i think this will happen:

1. You will update `.Values.hub.extraConfigMap` (the leftmost dict)
2. `$values.custom` will get lower priority than `.Values.hub.extraConfigMap`

I suggest:

```
{{- $_ := set $values "custom" (merge $values.custom .Values.hub.extraConfigMap) }}
# or, this may work as well
{{- $_ := merge $values.custom .Values.hub.extraConfigMap }}
```

Reply:
```
# this worked fine, NOTES.txt is a beauty as well
{{- $_ := merge $values.custom .Values.hub.extraConfigMap }}
```

Reply:
Good catch, I was thinking of Python priority, where the last item wins, not first. Fixed.